### DISCUSSION TITLE
Would love stronger filter options and list sharing

### DISCUSSION THREAD

**Discussion by DolfLund** [sct/overseerr#2697] [Feature Requests]
I find that the quality of a tool like this often lies in how easily you can clarify what your desires are. I wondered if you had any of these ideas already on your roadmap. 

- It's great to hide media already in my Plex library but if I could hide media that was in and/or watched, and if it was easier to toggle these preferences it would impact how I use the tool each time. 

- Filtering and sorting. If I could run a query of comedies and dramas that were rated above a certain rating that I have not watched and sort it by release date, this would be very powerful. And if we're going nuts, filtering could include any metadata like director, studio, live action or animated, etc

- List sharing, would be great if people had public watchlists we could share with each other and maybe follow, would be excellent to look through someone else's new suggestions, maybe even auto add those list contents to my requests (optional of course)

- Lastly I know this is a big one, but I love that Netflix uses my watch and rate history to suggest the match potential, while it's a HUGE idea, If I had the ability to rate items in Overseerr, and it could suggest items I would like based on similarity to other users shared tastes, it would be excellent. 

- I forgot if you can track multiple Plex libraries yet, but if you could also track libraries shared with you, then these suggestions could be a little more powerful as you could discover media already available to you, but not particularly attained by you. 

These are just some thoughts. I love what this team is doing and want to clarify that I am thankful and appreciative of project in it's current state.

**Comment by HAL9000000** [comment]
Filtering would be great 👍 with on /off options, especially : 
- movies / series 
- already downloaded / demanded
- above a specific rating level
- genre

**Comment by msb-glitch** [comment]
I would love some filtering options. The discover page of Overseer is almost useless to me because I watch very few blockbusters or franchises. Would love to be able to exclude things from showing up by studio (never show Marvel, for instance) or a wordlist. Would also love to be able to specifically include things on the discover page by studio ( I want to see every A24 release easily, for instance) or wordlist.

**Comment by HASSNypon30** [comment]
+1
I would settle to be enable to sort by year, movie, series, (anime?) on search,
If you search for something like "Batman" it would be SOOO much easier to find what you are looking for.

**Comment by seth100** [comment]
+1 for filtering options (and would be great to have also sorting options and more categories)

**Comment by blumarten** [comment]
any kind of sorting or filtering would be a blessing. overseer's almost useless to me as it's just a wall of mainstream stuff and there's no easy way to discover anything that's actually interesting to me.

**Comment by modem7** [comment]
I'd love to be able to even do something like:

Category: Animation
Sort by: Release year
